src/containers/orders/cart/filled-cart/filled-cart.js

```diff
@@ -14,32 +16,38 @@ const FilledCart = ({ items }) => {
     currency: Currency.currency
   }));
 
-  const totalPrice = items.reduce(
-    (acc, item) => acc + calcPrice(item, currency),
-    0
-  );
+  const totalPrice = items.reduce((acc, item) => acc + calcPriceForCart(item, currency), 0);
+
+  let currencySign;
+  switch (currency) {
```

Comment:
please use ternary operators and move to utils

Reply:
CI said, that ternary operator in ternary operator, is code smell, that why I use "switch".

Reply:
you should use ternary and move to utils